qadence/blocks/primitive.py

```diff
@@ -454,3 +454,34 @@ def _block_title(self) -> str:
 
         s += rf" \[params: {params_str}]"
         return s if self.tag is None else (s + rf" \[tag: {self.tag}]")
+
+
+class ProjectorBlock(PrimitiveBlock):
+    """The abstract ProjectorBlock."""
+
+    name = "ProjectorBlock"
+
+    def __init__(
+        self,
+        bra: str,
+        ket: str,
```

Comment:
Just a nitpick but I would reorder it as `ket, bra` so that `ProjectorBlock("00", "11")` means |00><11|.

Reply:
Done.